db/schema.rb

```diff
@@ -47,12 +47,11 @@
     t.date "average_views_updated_at"
     t.integer "wiki_id"
     t.integer "mw_page_id"
-    t.boolean "tracked", default: true
```

Comment:
I'm a little confused by this removal, as the migration exists for it.

Reply:
Issue resolved: the original migration said the column was type `bool`, which is not a valid type.